src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using System;
```

Comment:
`XmlDocumentationExtensions` remains a `static` class, so `IPathManager` has to be passed in.